ComputerVisionServer/moedx/client/remote_bench.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import os
+import json
+import time
+import logging
+import argparse
+import requests
+from threading import Thread
+
+LAUNCH_INTERVAL = 1 # Seconds
+# filename = "objects_320x180_x2.mp4"
+filename = "objects_320x180.mp4"
+# filename = "objects.mp4"
+
+def launch_remote_benchmark(url):
+    body = '-s 80.187.140.9 -e /object/detect/ -c websocket -f %s -n PING --server-stats' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c websocket -f %s -n PING --server-stats' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c websocket -f %s -n PING --fullsize' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c websocket -f %s -n PING' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c rest -f %s -n PING --skip-frames 4' %filename
```

Comment:
dead code?

Reply:
Keeping different versions handy to alter the tests.